### PR TITLE
Prefetching Core JS

### PR DESCRIPTION
**Pull request by dijs**

### CODE REVIEW

File: packages/react-storefront/test/Server.test.js [hunk 1]
```diff
@@ -79,6 +79,14 @@ describe('Server', () => {
       expect(exported.MOOV_PWA_RESPONSE.headers['content-type']).toBe('application/json')
     })
 
+    it('should set prefetch headers', async () => {
+      global.env.path = '/test'
+      request = new Request()
+      response = new Response(request)
+      await new Server({ theme, model, router, blob, globals, App }).serve(request, response)
+      expect(exported.MOOV_PWA_RESPONSE.headers.link).toBe('</pwa/>; rel=prefetch')
```
Comment (dijs) [inline packages/react-storefront/test/Server.test.js]: @markbrocato This was kinda weird. I don't know where this `pwa` value is coming from. I was hoping it is mocked somewhere as a chunk for testing.

File: README.md [hunk 2]
```diff
@@ -74,6 +74,10 @@ yarn release
 
 ## Changelog
 
+### 6.26.0
+
+- Server now prefetches core javascript sources using prefetch header
```
Comment (markbrocato) [inline README.md]:
```suggestion
- JavaScript bundles are now prefetched using link rel="prefetch" headers.
```

File: packages/react-storefront/src/Server.js [hunk 3]
```diff
@@ -155,6 +162,17 @@ export default class Server {
 
       const helmet = Helmet.renderStatic()
 
+      const chunks = flushChunkNames(stats)
+
+      // Set prefetch headers so that our scripts will be fetched
+      // and loaded as fast as possible
+      const prefetchHeaders = flatten([
```
Comment (markbrocato) [inline packages/react-storefront/src/Server.js]: I'd prefer to see the whole way we bring in scripts to be refactored.  Here you're duplicating operations to get the script URLs once for headers and again for tags.  There should just be one method that returns the URLs and then separate code that renders the headers and writes the script tags.

File: packages/react-storefront/test/Server.test.js (hunk 1 above)
Comment (markbrocato) [reply packages/react-storefront/test/Server.test.js]: It comes from react-storefront-stats.  Is that being mocked?

Comment (dijs) [reply packages/react-storefront/test/Server.test.js]: Yes it is. Thanks.

File: packages/react-storefront/src/Server.js (hunk 3 above)
Comment (dijs) [reply packages/react-storefront/src/Server.js]: Refactored.

**Pull request merged**